pages/about/groups/task-forces/matf.md

```diff
@@ -50,7 +50,7 @@ The Mobile A11y Task Force welcomes input from the global accessibility communit
 
 ### Contribute without joining the task force
 
-You can contribute to the work without being a member of the task force in the [Mobile A11y GitHub repository](https://github.com/w3c/Mobile-A11y-TF-Note/issues):
+You can contribute to the work without being a member of the task force in the [Mobile A11y GitHub repository](https://github.com/w3c/matf/):
```

Comment:
```suggestion
You can contribute to the work without being a member of the task force in the [Mobile A11y GitHub repository](https://github.com/w3c/matf/issues):
```

Feel free to dismiss my suggestion, but I just want to make sure you decided to point to the main page of the repo instead of the "Issues" page on purpose. The previous link lead to an "Issues" page, and the bullet points are about issues.

Reply:
Thanks... I did. The ask is to contribute not to raise issue... makes more sense to link to main page.